pywinauto/controls/HwndWrapper.py

```diff
@@ -1252,11 +1252,11 @@ def get_scroll_bar_info(self, scr_bar_object):
 
         scroll_bar_info = win32structures.PSCROLLBARINFO
         scroll_bar_info.cbSize = ctypes.sizeof(scroll_bar_info)
-        scroll_bar_info = win32functions.GetScrollBarInfo(self.handle, scr_bar_object)
-        if scroll_bar_info:
-            return True
+        msg = win32functions.GetScrollBarInfo(self.handle, scr_bar_object)
+        if msg == 0:
+            return None
```

Comment:
We must raise `RuntimeError` in case of failure, not just silently `return None`.